### ISSUE TITLE
Not loading on Stackoverflow, no SBS editor button

### ISSUE THREAD

**Issue by jcollum** [MarcinOrlowski/StackUnderflow#11]
See screenshot below, seems like there should be a "Toggle SBS" button in there but I don't see it.  

![image](https://cloud.githubusercontent.com/assets/704514/25597221/6f964f70-2e82-11e7-8c50-a5aa1aa63a93.png)

Tampermonkey 4.2.7 (up to date)

StackUnderflow is up to date according to Tampermonkey

**Comment by MarcinOrlowski** [comment]
When I disable Tampermonkey completely I see no SBS button either:

![su](https://cloud.githubusercontent.com/assets/8041294/25613251/dd5dfb38-2f2d-11e7-9453-63b80e03b952.png)

can you try the same? also check if you do not have any i.e. Stylish or other tools that may be messing with SO layout. AFAIR there's no code in StackUnderflow script that should affect SBS button at all.

**Comment by jcollum** [comment]
I uninstalled the extension, reinstalled extension and SU 1.1.0. Still not seeing the SU buttons. 

> can you try the same?

I can't get it to show, so disabling TM doesn't have any effect. 

> check if you do not have any i.e. Stylish or other tools that may be messing with SO layout

Don't have any. 


Here's the stacktrace: 

```
TypeError: Cannot read property 'split' of undefined
    at augmentQuestion (eval at E_c (:3:114), <anonymous>:10782:65)
    at Window.tms_f74677ff_ce95_4de6_a029_233ca22b8f7a (eval at E_c (:3:114), <anonymous>:10628:9)
    at <anonymous>:2:297
    at eval (eval at E_c (:3:114), <anonymous>:3:1)
    at eval (eval at E_c (:3:114), <anonymous>:4:3)
    at Object.eval (eval at E_c (:3:114), <anonymous>:11086:6)
    at <anonymous>:2:297
    at Object.E_c (<anonymous>:3:112)
    at E (eval at exec_fn (:1:108), <anonymous>:56:175)
    at Object.create (eval at exec_fn (:1:108), <anonymous>:64:282)
```

**Comment by MarcinOrlowski** [comment]
> I can't get it to show, so disabling TM doesn't have any effect.

Then it does not look like it's in any way related either to TM or SU.  I'm closing this. If you think otherwise, please explain why you reported it here.

**Comment by jcollum** [comment]
It should not be closed. Did you miss the stack trace above? `at augmentQuestion (eval at E_c (:3:114), <anonymous>:10782:65)` -- that's in your code. I can't get it to show because it's hitting an error in the augmentQuestion function, I assume.

**Comment by MarcinOrlowski** [comment]
Ah, indeed. I did miss that. But aside from that - I disabled SU and TM completely so there was no chance to interfere with SO. And I still have no button as I shown on the screenshot. My point is that you have the same issue (no button nonetheless) then this is most likely **separate** problem and is not related to TM/SU

**Comment by jcollum** [comment]
If your code is hitting a failure in `augmentQuestion` and failing to continue it sure seems like it's related to SU.

**Comment by MarcinOrlowski** [comment]
But if you disable TM/SU completely then it should be irrelevant how buggy it is, isn't it?

**Comment by jcollum** [comment]
It doesn't work when enabled. 

Not adding elements to SO when disabled is the expected behavior. I don't see the relevance.

**Comment by MarcinOrlowski** [comment]
Hold on :) You reported that SBS button is gone due to SU. The stacktrace you quoted indeed indicates something may be wrong and this report is accepted. What I am talking next, is that I see **no** SBS button even if I completely disable TM/SU.  You reported `Not loading on Stackoverflow, no SBS editor button` and  for me it looks now that `Not loading on Stackoverflow` is relevant to SU, but the `no SBS editor button` part seems not, because the SBS should be there once TM/SU is off (as at that point there should be no chance for TM/SU to interfere) but my test shown SBS is not visible (see screenshot above) nonetheless. That's why I asked if you actually can see SBS button with TM/SU **off** because if not, then that part is not related. If you do, then in fact the problem shown in stacktrace can be causing that (but then the question is why I do not see SBS here?). Do I miss something?

**Comment by jcollum** [comment]
> because the SBS should be there once TM/SU is off as at that point

I'm lost. Does SU create an SBS editor button on the "ask question" page? It used to. Has that been changed?

**Comment by MarcinOrlowski** [comment]
No, SBS button is not part of SU. Maybe you meant this project https://github.com/soscripted/sox as it seem to have something like that.

**Comment by jcollum** [comment]
Right, sorry, I was thinking of SOX, not Stackunderflow. Memory failure. Well, I guess the bug report is legit anyway. :-/
